.github/workflows/tests.yml

```diff
@@ -34,23 +33,74 @@ jobs:
       uses: actions-rs/toolchain@v1
       with:
         profile: minimal
-        toolchain: nightly-2021-02-14
-        components: rustfmt, clippy
+        toolchain: nightly-2021-02-22
         target: wasm32-unknown-unknown
         override: true
         default: true
 
-    # Enable this for clippy linting.
-    # - name: Check and Lint Code
-    #   run: cargo +nightly-2021-02-14 clippy -- -D warnings
     - name: Check Code
       run: cargo check --all --all-features
```

Comment:
maybe we can just run the tests? Could be faster? Not sure but maybe worth a try?

Reply:
Definitely, fixed in fd30b5a